cfg/Movebasic.cfg

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python2
+PACKAGE = "move_basic"
+
+import roslib; roslib.load_manifest(PACKAGE)
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+gen.add("max_turning_velocity", double_t, 0, "Max turning velocity (when turning on spot)", 1.0,  0, 4.0)
+gen.add("max_lateral_velocity", double_t, 0, "Max lateral velocity (when going straight)", 0.5,  0, 4.0)
+gen.add("angular_acceleration", double_t, 0, "Angular acceleration", 0.3,  0.01, 4.0)
+gen.add("max_linear_velocity", double_t, 0, "Max linear velocity", 0.5,  0, 1.1)
+gen.add("linear_acceleration", double_t, 0, "Linear_acceleration", 0.25,  0, 1.1)
+gen.add("angular_tolerance", double_t, 0, "Angular tolerance (when turning on spot)", 0.01,  0, 0.5)
+gen.add("linear_tolerance", double_t, 0, "Linear tolerance", 0.1,  0, 0.2)
+gen.add("lateral_kp", double_t, 0, "Lateral P", 2.0,  0, 20.0)
+gen.add("lateral_ki", double_t, 0, "Lateral I", 0.0,  0, 20.0)
+gen.add("lateral_kd", double_t, 0, "Lateral D", 20.0,  0, 20.0)
+gen.add("linear_gain", double_t, 0, "Linear velocity gain", 1.0, 0, 10.0)
+gen.add("rotational_gain", double_t, 0, "Angular velocity gain", 2.5, 0, 10.0)
+gen.add("velocity_threshold", double_t, 0, "Velocity threshold (to mark goal as finished)", 0.1, 0, 0.5)
```

Comment:
Maybe also add units into description - here m/s - also holds for other relevant params

Reply:
Will do, thanks